src/css/modal.styl

```diff
@@ -1,9 +1,10 @@
 .modal
   &-container-md
     background-color: $white
-    min-width: 45vw;
+    @media screen and (max-width: 767px)
+      min-width: 65vw;
     @media screen and (min-width: 768px)
-      max-width: 65vw
+      width: 45vw;
```

Comment:
Refacto de l'affichage des modales, pour avoir une taille fixe sur grand écran. J'ai testé sur les modales d'ajout et de modification d'un créneau et le résultat ne me perturbe pas. Qu'en pensez-vous ?

Reply:
normalement tu peux le simplifier
```
&-container-md 
    background-color: $white
    min-width: 65vw;
    @media screen and (min-width: 768px)
      width: 45vw;
```

Reply:
Je ne pense pas parce que je veux deux propriétés différentes selon si l'écran est plus ou moins grand que 768 px.
Là j'ai un comportement étrange.
Et avec l'inverse : 
`  &-container-md
    background-color: $white
    width: 45vw;
    @media screen and (max-width: 767px)
      min-width: 65vw;
`
le comportement est moins agréable je trouve.

Qu'en penses-tu ?